data/repo_meta/new_rust.txt

```diff
@@ -1,51 +1,86 @@
+tafia/quick-xml
```

Comment:
why are we renaming `rust.txt` to `new_rust.txt`

Reply:
They're all just going to be `<language>.txt` from now on since we're going to merge the lists together.